ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -37,6 +37,21 @@ function prepare_next_steps() {
       "${dir}/auth/kubeadmin-password" \
       "${dir}/metadata.json"
 
+  # For private cluster, the bootstrap address is private, installer cann't gather log-bundle directly even if proxy is set
+  # the workaround is gather log-bundle from bastion host
+  # copying install folder to bastion host for gathering logs
+  set -x
+  publish=$(grep "publish:" ${SHARED_DIR}/install-config.yaml | awk '{print $2}')
+  if [[ "${publish}" == "Internal" ]] && [[ ! $(grep "Bootstrap status: complete" "${dir}/.openshift_install.log") ]]; then
+    echo "Copying install dir to bastion host."
+    bastion_ssh_user=$(head -n 1 "${SHARED_DIR}/bastion_ssh_user")
+    bastion_public_address=$(head -n 1 "${SHARED_DIR}/bastion_public_address")
+    cmd="scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o VerifyHostKeyDNS=yes  -i \"${CLUSTER_PROFILE_DIR}/ssh-privatekey\" -r ${dir} ${bastion_ssh_user}@${bastion_public_address}:/tmp/"
```

Comment:
The same comments as https://github.com/openshift/release/pull/33912#discussion_r1018810934

Reply:
updated.